src/main/cms/cms_types.h

```diff
@@ -58,11 +58,11 @@ typedef const void *(*CMSEntryFuncPtr)(displayPort_t *displayPort, const void *p
 
 typedef struct
 {
-    const char * const text;
-    const OSD_MenuElement type;
+    const char * text;
```

Comment:
What's the reason to remove these `const`s?

Reply:
Because the values can’t be set at runtime if they’re `const`. The normal menus are defined as a `const` array of entries anyway.